osmaxx/excerptexport/_settings.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+from django.conf import settings
+
+POLYFILE_LOCATION = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data', 'polyfiles')
```

Comment:
I think I'd find this easier to parse if `abspath` was on the outest-most nesting level:
```python
POLYFILE_LOCATION = os.path.abspath(os.path.join(os.path.dirname(__file__), 'data', 'polyfiles'))
```
(resulting value should be the same, if I'm not mistaken)